xapian-applications/omega/omindextest

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
+# omindextest: Test omindex
+#
+# Copyright (C) 2019 Bruno Baruffaldi
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301
+# USA
+
+DIR=$(pwd)
+TEST_FILES="$DIR/testfiles/"
+TEST_DB="$DIR/testdatabase"
+
+./omindex --db $TEST_DB --url $TEST_FILES $TEST_FILES
```

Comment:
I'd suggest `--url=/` so that the terms in the database produced don't depend on the directory the source code is checked out in.